Makefile

```diff
@@ -1,15 +1,22 @@
-ROOT_DEST=/opt/timecard-app
+
+APP_NAME_LOWER=timecard-app
+APP_NAME_UPPER=Timecard-App
+
+ROOT_DEST=/opt/${APP_NAME_LOWER}
 ROOT_BIN=/usr/bin
 ROOT_APPS=/usr/share/applications/
 ROOT_ICONS=/usr/share/icons/hicolor/scalable/apps/
 
-USER_DEST=/home/${USER}/.timecard-app
+USER_DEST=/home/${USER}/.${APP_NAME_LOWER}
 USER_BIN=/home/${USER}/.local/bin/
 USER_APPS=/home/${USER}/.local/share/applications/
 USER_ICONS=/home/${USER}/.icons/hicolor/scalable/apps/
 
-DESKTOP_FILE=com.indeliblebluepen.timecard_app.desktop
-ICON_FILE=timecard.svg
+SHARE_PATH=share
+DESKTOP_FILE_PATH=${SHARE_PATH}/applications
+DESKTOP_FILE=com.codemouse92.timecard.desktop
+ICON_FILE_PATH=${SHARE_PATH}/icons
```

Comment:
I like this change as a whole, but I think the names should be `RESOURCE_SOURCE_PATH`, `DESKTOP_FILE_SOURCE_PATH`, and `ICON_FILE_SOURCE_PATH` to avoid confusion. When I first saw the names, I was expecting this to be the *destination*.

That said, I can merge and change that myself.